serverless/cli_tools/kn-plugins.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="kn-plugins"]
+= Knative CLI plug-ins
+include::_attributes/common-attributes.adoc[]
+:context: kn-plugins
+
+toc::[]
+
+The Knative (`kn`) CLI supports the use of plug-ins, which enable you to extend the functionality of your `kn` installation by adding custom commands and other shared commands that are not part of the core distribution. Knative (`kn`) CLI plug-ins are used in the same way as the main `kn` functionality.
+
+Currently, Red Hat supports the `kn-source-kafka` plug-in and the `kn event` plug-in.
```

Comment:
```suggestion
Currently, Red Hat supports the `kn-source-kafka` plug-in and the `kn-event` plug-in.
```

Maybe add hyphen, to be in line with `kn-source-kafka` or remove them there as well?!?